src/d3a/d3a_core/sim_results/stats.py

```diff
@@ -82,6 +84,7 @@ def _store_sold_trade(cls, result_dict, trade):
         result_dict['earned'] += (trade.offer.price - fee_price) / 100.
         result_dict['total_energy'] -= trade.offer.energy
         result_dict['total_cost'] -= (trade.offer.price - fee_price) / 100.
+        result_dict['market_fee'] += fee_price / 100.0
```

Comment:
This change has to be decided in D3ASIM-2303

Reply:
Will remove it from here.